uniprot-search/src/main/resources/crossref/search-fields.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
General comment, applicable to all json file.
1. How are we capturing the SearchFiledType "TERM" or "RANGE"?
2. How are capturing the float value  boost?

Reply:
1. E.g., in UniProtKB's search-field.json, we have: `"rangeField": "ftlen_metal",`
2. Boosts will be extracted to a separate boost file, as done with UniProtKB, so that we have finer control, and can generate correct queries according to boosts (current way doesn't always generate correct queries, as we found out for KB's result ordering analysis).